### ISSUE TITLE
Error when deploying for the first time with certain stage names

### ISSUE THREAD

**Issue by goto3** [serverless-stack/serverless-stack#1790]
Hello, when deploying for the first time in a directory it asks for a stage name:
`Look like you’re running sst for the first time in this directory. Please enter a stage name you’d like to use locally. Or hit enter to use the one based on your AWS credentials`

If you enter a stage name with spaces it throws this error:

```
Preparing your SST app

=======================
 Deploying debug stack
=======================

node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\goto\Documents\Proyectos\Proyecto\sst\examples\react-app\node_modules\@serverless-stack\cli\assets\debug-stack\us-east-1\.build\lib\index.js' imported from C:\Users\goto\Documents\Proyectos\Proyecto\sst\examples\react-app\node_modules\@serverless-stack\cli\assets\debug-stack\bin\index.mjs
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ESMLoader.import (node:internal/modules/esm/loader:380:22)
    at importModuleDynamically (node:internal/modules/esm/translators:106:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
    at file:///C:/Users/goto/Documents/Proyectos/Proyecto/sst/examples/react-app/node_modules/@serverless-stack/cli/assets/debug-stack/bin/index.mjs:21:51 {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

if I set a stage name a word with no spaces this does not happen.
It would be nice if it showed a nicer error message or better yet, replace spaces with `-`, I spent like 30 minutes figuring what it was.

Thanks

**Comment by fwang** [comment]
Hey @goto3, this is fixed in [1.2.20](https://github.com/serverless-stack/serverless-stack/releases/tag/v1.2.20).

Nice catch!

**Comment by goto3** [comment]
Thank you!
